src/app/containers/TopStories/index.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useContext } from 'react';
```

Comment:
With a couple of small changes, this container can be easily generalised to be re-usable for any other CPS Group in your secondary column.
If `labelId` and `data` are passed in as props & you rename `TopStoriesWrapper` to be `Wrapper`.

Then this container can be namespaced to be, for example, `src/app/containers/CpsAssetRelatedStories/index.jsx`.
A benefit of this is that it can be clear that this container is *only* for CPS Assets and not for Front Pages or other CPS Index pages.

Reply:
Hi Sareh. [This issue](https://github.com/bbc/simorgh/issues/5795) already tracks refactoring the OJ components that make use of StoryPromo, so we don't repeat code.

Reply:
I've renamed the files, so its folder is now `CpsTopStories`.